### ISSUE TITLE
Assets precompile error

### ISSUE THREAD

**Issue by Tamiyadd** [shakacode/react_on_rails#435]
Hi @justin808, i upgraded my react on rails version from 6.0.0.rc1 to 6.0.0 but i started to get an error on assets precompile. 

When precompiling assets it throws the following error:

```
TypeError: no implicit conversion of nil into String
.rvm/gems/ruby-2.1.2/gems/react_on_rails-6.0.0/lib/tasks/assets.rake:30:in `initialize'
.rvm/gems/ruby-2.1.2/gems/react_on_rails-6.0.0/lib/tasks/assets.rake:30:in `new'
.rvm/gems/ruby-2.1.2/gems/react_on_rails-6.0.0/lib/tasks/assets.rake:30:in `block (3 levels) in <top (required)>'
.rvm/gems/ruby-2.1.2/gems/react_on_rails-6.0.0/lib/tasks/assets.rake:109:in `block in <top (required)>'
.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `eval'
.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => react_on_rails:assets:symlink_non_digested_assets
(See full trace by running task with --trace)

```

i investigated in that line of code and found that the gem search for `.sprockets-manifest-*.json` manifest file but i have a `manifest-*.json` manifest file. 
Maybe the problem is there? i don't know why with the rc1 everything works well and with this version it's not working

can you help me?

thanks

**Comment by justin808** [comment]
@alleycat-at-git @Tamiyadd  Ahh! this must be related to the sprockets version.

We need to check for that and maybe allow either file.

Let me know on the sprockets version and we'll try to get this fixed asap. Also, can you post the contests of your manifest-*.json file?

**Comment by alleycat-at-git** [comment]
@Tamiyadd, could you please try [this PR](https://github.com/shakacode/react_on_rails/pull/436) out? Did it fix the issue?

**Comment by Tamiyadd** [comment]
@alleycat-at-git i tried your PR and it works like a charm! 

for your info i'm using sprockets `2.12.4`,
do you need my manifest.json content anyway?

**Comment by alleycat-at-git** [comment]
Cool! I think we now that you checked it worked, we don't need the file anymore.

**Comment by justin808** [comment]
Released in 6.0.1.
